dluhc-component-library/src/index.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+  font-family: "GDS Transport", arial, sans-serif;
```

Comment:
The GDS Transport font doesnt currently work as its not included. This will be fixed in another PR